djed-sdk/src/web3.js

```diff
@@ -1,12 +1,35 @@
 import Web3 from "web3";
 import djedArtifact from "./artifacts/DjedABI.json";
 import coinArtifact from "./artifacts/CoinABI.json";
+import oracleArtifact from "./artifacts/OracleABI.json";
 import { convertInt,web3Promise } from "./helpers";
 
-export const DjedInstance = async (BLOCKCHAIN_URI, DJED_ADDRESS) => {
+ const getWeb3 = (BLOCKCHAIN_URI) =>
+  new Promise(async (resolve, reject) => {
+    if (window.ethereum) {
+      try {
+        const web3 = new Web3(BLOCKCHAIN_URI);
+        resolve(web3);
+      } catch (error) {
+        reject(error);
+      }
+    } else {
+      reject("Install Metamask");
+    }
+  });
+
+  const getOracleContract = (web3, oracleAddress, msgSender) => {
```

Comment:
Can we move this to `system.js`?

Reply:
Or maybe `oracle.js` ?

Reply:
@ceilican This function is required initially to fetch the contract,we can move this to djed.js?

Reply:
OK. `djed.js` is already a better location than `web3.js`. Let's move it.